### PR TITLE
adding inheritance filter

### PR DESCRIPTION
**Pull request by shawn-hurley**
Signed-off-by: Shawn Hurley <smhurley00@gmail.com>

### CODE REVIEW

**Comment by shawn-hurley** [comment]
Please wait for https://github.com/konveyor/java-analyzer-bundle/pull/23 before merging

File: demo-rules/local-storage.windup-rewrite.yaml [hunk 1]
```diff
@@ -18,30 +18,36 @@
     # - java.referenced:
     #     location: CONSTRUCTOR_CALL
     #     pattern: java.io.FileInputStream(*)
+    # - java.referenced:
```
Comment (fabianvf) [inline demo-rules/local-storage.windup-rewrite.yaml:21]: Is this intentional?

File: windup-testing.Dockerfile [hunk 2]
```diff
@@ -1,6 +1,7 @@
-FROM quay.io/konveyor/jdtls-server-base
+FROM a0dd63e0613c295f334e27956aa0f659835728b2c2d0f1dfacdbfcb52f34abf4
```
Comment (fabianvf) [inline windup-testing.Dockerfile:1]: Is this intentional?

Comment (shawn-hurley) [reply windup-testing.Dockerfile:1]: nope good cache!

File: demo-rules/local-storage.windup-rewrite.yaml (hunk 1 above)
Comment (shawn-hurley) [reply demo-rules/local-storage.windup-rewrite.yaml:21]: Also nope!

**Review by fabianvf — APPROVED** [review]
This is awesome

**Pull request merged**